lua/starfall/libs_cl/render.lua

```diff
@@ -471,7 +473,10 @@ end)
 
 
 function instance:prepareRender()
-	currentcolor = COLOR_WHITE
+	rawset(currentcolor, "r", rawget(currentcolor, "r"))
+	rawset(currentcolor, "g", rawget(currentcolor, "g"))
+	rawset(currentcolor, "b", rawget(currentcolor, "b"))
+	rawset(currentcolor, "a", rawget(currentcolor, "a"))
```

Comment:
Should be setting to 255, this code currently isn't doing anything.

Reply:
Damnnn